hugegraph-core/src/main/java/org/apache/hugegraph/backend/store/raft/compress/CompressStrategyManager.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.hugegraph.backend.store.raft.compress;
+
+import org.apache.hugegraph.config.CoreOptions;
+import org.apache.hugegraph.config.HugeConfig;
+
+public class CompressStrategyManager {
+    private static CompressStrategy[] compressStrategies = new CompressStrategy[5];
```

Comment:
can we define a const var for the 5 in `new CompressStrategy[5]`, like `new CompressStrategy[MAX_STRATEGY] `and move it to line 11.